be/src/io/cache_input_stream.cpp

```diff
@@ -463,4 +461,11 @@ void CacheInputStream::_populate_cache_from_zero_copy_buffer(const char* p, int6
     return;
 }
 
+bool CacheInputStream::_can_ignore_populate_error(const Status& status) const {
```

Comment:
`is_cancelled` will no logger processed?

Reply:
> is_cancelled

Yes, this is a deprecated status code and will not happen now.